conf/defaults.ini

```diff
@@ -939,6 +939,26 @@ admin_config_poll_interval = 60s
 # The interval string is a possibly signed sequence of decimal numbers, followed by a unit suffix (ms, s, m, h, d), e.g. 30s or 1m.
 alertmanager_config_poll_interval = 60s
 
+# The redis server address that should be connected to.
+ha_redis_address =
```

Comment:
I think we usually also copy new config keys into `sample.ini` and comment it out.